tests/setup_cfg_fmt_test.py

```diff
@@ -97,6 +99,8 @@ def test_noop(tmpdir):
             '    req12\n'
             '    req13!=2,>=7\n'
             '    req14>=1,<=2\n'
+            '    req15@ git+https://github.com/foo/bar.git@master\n'
```

Comment:
this spacing seems weird, does it work without a space afterwards? that would be more in line with everything else (and an easier patch, just add @ as an operator)

Reply:
Agreed on the spacing; it was just to look like `pip` output, but if that's not a priority, I like the space-less look better. (It works either way.)

Absolutely correct on adding it as an operator being easier. I tried that first but didn't realize I needed to change both regexes. Much better now :)